packages/language-server/src/commands/ExecuteCommandProvider.ts

```diff
@@ -3,34 +3,60 @@ import { isInsideHtmlRegion } from '../utils/node';
 import { DocumentCache } from '../documents';
 
 export enum Command {
+    DebugCommand = 'twiggy.debug-command',
     IsInsideHtmlRegion = 'twiggy.is-inside-html-region',
 }
 
-const commands = new Map([
-    [Command.IsInsideHtmlRegion, isInsideHtmlRegion],
-]);
-
 export class ExecuteCommandProvider {
+    private commands: Map<Command, (...args: any) => Promise<any>>;
+
     constructor(
         connection: Connection,
         private readonly documentCache: DocumentCache,
     ) {
+        this.commands = new Map<Command, (...args: any) => Promise<any>>([
+            [Command.DebugCommand, this.debugCommand],
+            [Command.IsInsideHtmlRegion, this.isInsideHtmlRegionCommand],
+        ]);
+
         connection.onExecuteCommand(
             this.onExecuteCommand.bind(this),
         );
     }
 
     async onExecuteCommand(params: ExecuteCommandParams) {
-        const [ commandName ] = params.command.split('(');
-        const command = commands.get(commandName as Command);
+        const command = this.commands.get(params.command as Command);
 
-        if (!command || !params.arguments) {
+        if (
+            !command ||
+            !params.arguments ||
+            params.arguments.length < 1 ||
+            'string' !== typeof params.arguments[0]
+        ) {
             return;
         }
 
-        const [uri, position] = params.arguments as [DocumentUri, Position];
-        const document = await this.documentCache.get(uri);
+        const [ workspaceFolderPath, ...args ] = params.arguments as [string, ...any];
+        if (workspaceFolderPath !== this.documentCache.workspaceFolderPath) {
```

Comment:
The only bug left here is that lsp won't start with multi-folder workspace setup.

quick repro:
1. `cp your-twig-project your-twig-project-copy`
1. vscode: open your-twig-project
1. File -> Add folder to Workspace -> `your-twig-project-copy`

expected: works in both \
actual: can't register `twiggy.is-inside-html-region` twice